xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1083,9 +1085,7 @@ void CRenderManager::PrepareNextRender()
   double frameOnScreen = m_dvdClock.GetClock();
   double frametime = 1.0 / g_graphicsContext.GetFPS() * DVD_TIME_BASE;
 
-  // correct display latency
-  // internal buffers of driver, assume that driver lets us go one frame in advance
-  double totalLatency = DVD_SEC_TO_TIME(m_displayLatency) - DVD_MSEC_TO_TIME(m_videoDelay) + 2* frametime;
```

Comment:
Hm if we move this into GraphicsContext anyway, would you agree that it makes sense to move the `g_advancedSettings.GetDisplayLatency()` call which applies the overrides from advancedsettings to GraphicsContext, too?

> I was thinking more about usage of g_windowing and I am inclined to drop direct use of this global in general.

Yeah most places shouldn't need to use it directly. You have to consider that g_Windowing is *also* the RenderSystem instance which is used as such all over the code though. Maybe that should be separated anyway.